packages/internals/src/__tests__/getGenerators/generatorResolvers/prisma-client-js.test.ts

```diff
@@ -0,0 +1,12 @@
+import { semverLt } from '../../../get-generators/generatorResolvers/prisma-client-js/check-dependencies/semverLt'
+
+describe('compare semverLt as numbers', () => {
+  test('compare semverLt version is working correct', () => {
+    // compare semverLt as Number not as Strings. '5' > '10' is true in Strings
+    const minVersion = '1.1.10'
+    const testVersion = '1.1.5'
+    // semverLt returns true if currentVersion is lower than the minVersion
+    const results = semverLt(testVersion, minVersion)
+    expect(results).toBe(true)
+  })
```

Comment:
I think we could test a little more in general to check this function works as expected (as this is also its first test). Not a requirement though, I'll leave it up to you :)